### PR TITLE
Add SDSS gri filters for LATISS

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

**Comment by patrickingraham** [comment]
CI run is:
https://ci.lsst.codes/blue/organizations/jenkins/stack-os-matrix/detail/stack-os-matrix/34975/pipeline

File: python/lsst/obs/lsst/filters.py [hunk 1]
```diff
@@ -226,6 +226,17 @@ def addFilter(filter_dict, band, physical_filter, lambdaEff=0.0):
                      lambdaEff=0.0,
                      band="r",
                      afw_name="rg"),
+    FilterDefinition(physical_filter="SDSSg",
+                     lambdaEff=477.0,
+                     band="g"),
+    FilterDefinition(physical_filter="SDSSr",
+                     lambdaEff=623.1,
+                     band="r",
+                     afw_name="r"),
```
Comment (timj) [inline python/lsst/obs/lsst/filters.py:235]: Can you try it without the afw_name?

Comment (patrickingraham) [reply python/lsst/obs/lsst/filters.py:235]: uhh I thought I force pushed that change and re-tested... i'll follow up.

Comment (patrickingraham) [reply python/lsst/obs/lsst/filters.py]: ugh.. ya I force pushed but apparently didn't add the change :(  I'll re-run the CI. Sorry.

**Review by timj — APPROVED** [review]

**Comment by patrickingraham** [comment]
re-ran CI with changes successfully
https://ci.lsst.codes/blue/organizations/jenkins/stack-os-matrix/detail/stack-os-matrix/34978/pipeline

**Pull request merged**